src/App.tsx

```diff
@@ -85,6 +87,9 @@ const App: React.FC = () => {
                       <Route path={Routes.TEAM_LEADERBOARD}>
                         <Redirect to={Routes.LOGIN} />
                       </Route>
+                      <Route path={Routes.AVAILABLE_TEAMS}>
+                        <Redirect to={Routes.LOGIN} />
+                      </Route>
```

Comment:
https://github.com/Code-4-Community/speak-for-the-trees-frontend/pull/42/files#r585886582

Same comment as Sofie's PR, just something to think about.